packages/backend/server/src/plugins/copilot/resolver.ts

```diff
@@ -339,18 +375,27 @@ export class CopilotResolver {
     @Args('docId', { nullable: true }) docId?: string,
     @Args('options', { nullable: true }) options?: QueryChatSessionsInput
   ): Promise<CopilotSessionType[]> {
-    if (!copilot.workspaceId) return [];
+    if (!copilot.workspaceId) {
+      throw new NotFoundException('Workspace not found');
+    }
     await this.ac
       .user(user.id)
       .workspace(copilot.workspaceId)
       .allowLocal()
       .assert('Workspace.Copilot');
-    return await this.chatSession.listSessions(
+    const sessions = await this.chatSession.listSessions(
       user.id,
       copilot.workspaceId,
       docId,
       options
     );
+    return sessions.map(session => ({
+      id: session.sessionId,
+      parentSessionId: session.parentSessionId,
+      promptName: session.prompt.name,
+      model: session.prompt.model,
+      optionalModels: session.prompt.optionalModels,
+    }));
```

Comment:
this logic can share with `session` function in a private function

Reply:
Done